dscgt/DSCGT-Website
# Spelling ui changes

### Description
Some minor UI changes and one bugfix. Modified homepage image to make "Clubs" singular; fixed word overflow in Projects section of homepage; fixed a navbar link on Teams page.

### How to Test
1. Clone this branch
1. Start a local instance of the site with `dev_appserver.py app.yaml `
    1. If this is not working for you, make sure you've followed instructions under this project's README's "Installing Google Cloud SDK" section.
1. navigate to localhost:8080 and verify that the mentioned changes in the homepage and the team page have been applied
